Chart: Update Pyramid Chart demo description (#30690)
DevExpress/DevExtreme: apps/demos/Demos/Charts/PyramidChart/description.md

> DevExtreme Funnel can display data in a pyramid chart. To configure the component as a pyramid chart, assign *"dynamicHeight"* to the [algorithm](/Documentation/ApiReference/UI_Components/dxFunnel/Configuration/#algorithm) property and enable [inverted](/Documentation/ApiReference/UI_Components/dxFunnel/Configuration/#inverted).

Reviewer: I am shortening this paragraph and resolve some clarity issues:

DevExtreme Funnel can display data as a pyramid chart. To create such visualization, assign *"dynamicHeight"* value to the [algorithm](/Documentation/ApiReference/UI_Components/dxFunnel/Configuration/#algorithm) property and [invert the Funnel](/Documentation/ApiReference/UI_Components/dxFunnel/Configuration/#inverted).

Author: I would only like to remove "value":
```suggestion
DevExtreme Funnel can display data as a pyramid chart. To create such visualization, assign "dynamicHeight" to the [algorithm](/Documentation/ApiReference/UI_Components/dxFunnel/Configuration/#algorithm) property and [invert the Funnel](/Documentation/ApiReference/UI_Components/dxFunnel/Configuration/#inverted).
```

Reviewer: I think that works.

Although with a fresh eye, I can suggest another variant. See if this works:
DevExtreme Funnel can display data as a pyramid chart. To create such visualization, set the algorithm property to dynamicHeight and invert the Funnel.

<!--split-->

> This demo specifies multiple Funnel configuration objects:

Reviewer: This demo configures the following objects:

- [title](/Documentation/ApiReference/UI_Components/dxFunnel/Configuration/title/)    

> Specifies a title for the Funnel component.

Reviewer: Specifies the component title.

- [tooltip](/Documentation/ApiReference/UI_Components/dxFunnel/Configuration/tooltip/)    
Specifies item tooltips.
- [item](/Documentation/ApiReference/UI_Components/dxFunnel/Configuration/item/)    

> Customizes the appearance of items.

Reviewer: Customizes item appearance.

> Customizes the visual appearance of items.

Reviewer 2: ```suggestion
Customizes the appearance of items.
```
